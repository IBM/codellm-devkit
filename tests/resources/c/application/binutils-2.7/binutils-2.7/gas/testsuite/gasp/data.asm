
foo	.DATA	1,2,3
bar	.DATA	1,2,3,4,5 ,6
	.DATA.B	12345,12,2
	.DATA.W 9,2,12,3,13+41,foo+9
	.DATA.L 2~99



	.DATAB	1,2,3
	.DATAB	1,2,3




	.DATAB	1,9+32

	.DATAB.L	1,H'11111111
	.DATAB.W	2,H'2222
	.DATAB.B	3,H'333


	.END
