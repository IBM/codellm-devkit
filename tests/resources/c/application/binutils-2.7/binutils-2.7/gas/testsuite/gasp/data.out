!

!foo	.DATA	1,2,3
foo:	.long	1,2,3
!bar	.DATA	1,2,3,4,5 ,6
bar:	.long	1,2,3,4,5,6
!	.DATA.B	12345,12,2
	.byte	12345,12,2
!	.DATA.W 9,2,12,3,13+41,foo+9
	.short	9,2,12,3,54,foo+9
!	.DATA.L 2~99
	.long	97
!

!

!

!	.DATAB	1,2,3
	.fill	1,4,2
!	.DATAB	1,2,3
	.fill	1,4,2
!

!

!

!

!	.DATAB	1,9+32
	.fill	1,4,41
!

!	.DATAB.L	1,H'11111111
	.fill	1,4,286331153
!	.DATAB.W	2,H'2222
	.fill	2,2,8738
!	.DATAB.B	3,H'333
	.fill	3,1,819
!

!

!	.END
